modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -281,6 +302,63 @@ typedef struct indigo_core_gentable_ops {
         void *table_priv, void *entry_priv, of_list_bsn_tlv_t *key,
         of_desc_str_t err_txt);
 
+    /**
+     * @brief Add an entry
+     * @param op_ctx operation context. These data will be used when resume
+     *               function is called in the async operation. 
+     * @param table_priv Table private data
+     * @param key Entry key (identical to key from add)
+     * @param value New entry value
+     * @param [out] entry_priv Opaque private data for the entry, passed back
+     *                         whenever another operation is made on the entry.
+     *                         For async operation (return INDIGO_ERROR_PENDING)
+     *                         this value assigned to entry_priv will be
+     *                         ignored by state manager. The entry_priv should
+     *                         be assigned in resume API.
```

Comment:
"... will be ignored by state manager.  Instead, the entry_priv passed to the resume function will be stored if the asynchronous operation is successfully completed by the lower-level driver"